handlers/handler-router/src/main/java/org/apache/servicecomb/router/custom/RouterInvokeFilter.java

```diff
@@ -71,9 +71,6 @@ public Response afterReceiveRequest(Invocation invocation,
     if (!isHaveHeadersRule()) {
       return null;
     }
-    if (!RouterRuleCache.isServerContainRule(invocation.getMicroserviceName())) {
-      return null;
-    }
```

Comment:
It's recommended to add a unit test case that cover the logic of RouterInvokFilter, RouterServerListFilter. Adding this can help to make sure this module works as expected. Now there is no intergration test for this module.